.github/workflows/frontend.yml

```diff
@@ -0,0 +1,96 @@
+name: Frontend
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'release**'
+      - 'feature**'
+      - 'fix**'
+      - 'ci**'
+    tags:
+      - '**'
+    paths:
+    - 'frontend/**'
+    - 'enterprise/frontend/**'
+    - '**/yarn.lock'
+    - '.github/workflows/**'
+
+jobs:
+
+  fe-linter-flow:
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
```

Comment:
Given that linters can break the whole branch and they don't take too long, I'd maybe leave them ON even if someone provides `[ci skip]`.
This scenario happened to me in a `release-x.37.x` branch. I edited one line of code directly in GitHub and that backfired because prettier wasn't happy :)

Reply:
We can definitely do that, particularly since this workflow is **already conditional** anyway upon some commits touching `frontend/`.

Reply:
I vote for that.
Positives (confidence) outweight the nagatives (couple of minutes of CI checks regardles of a skip flag).